docs/streaming_endpoints.md

```diff
@@ -0,0 +1,50 @@
+# Streaming API (preview) {#ovms_docs_streaming_endpoints}
+
+## Introduction
+OpenVINO Model Server implements gRPC KServe extension which adds separate RPC for [bidirectional streaming](https://grpc.io/docs/what-is-grpc/core-concepts/#bidirectional-streaming-rpc) use cases. It means that besides unary RPC, where client sends a single request and gets back a single response, client is able to initiate connection and send/receive messages in any number and order using `ModelStreamInfer` procedure:
+
+```
+service GRPCInferenceService
+{
+  ...
+  // Standard
+  rpc ModelInfer(ModelInferRequest) returns (ModelInferResponse) {}
+  // Extension
+  rpc ModelStreamInfer(stream ModelInferRequest) returns (stream ModelStreamInferResponse) {}
+  ...
+}
+```
+
+This becomes very useful for endpoints serving [MediaPipe Graphs](./mediapipe.md). In unary inference RPC each request initializes new MediaPipe graph, inputs are deserialized into MediaPipe packets and pushed into the graph. After the graph is done, it is deleted.
+However, in streaming inference RPC MediaPipe graph is created only once per connection, and is reused by subsequent requests to the same gRPC stream. This removes graph initialization overhead and increases its overall throughput.
+
+![diagram](streaming_diagram.svg)
+
+## Graph Selection
+Opening stream is not enough to create execution graph. Instead, the first gRPC request after the opening defines which graph definition will be selected for execution (`model_name` and `model_version` proto fields).
+Afterwards, subsequent requests are required to match the servable name and version, otherwise the error is reported and input packets are not pushed to the graph. However, the graph remains in available for correct requests.
+
+> NOTE: The server closes the stream after the first request if requested graph is non-existent or retired.
```

Comment:
add info what happens if new requests have incorrect graph name and when in other cases the session is closed by the server

Reply:
There is information what happens, when subsequent (2+) request has incorrect name and version.

> Afterwards, subsequent requests are required to match the servable name and version...

Isn't it enough?

Reply:
that is enoguh, in which other cases the session will be closed?

Reply:
The stream is closed when graph encounters an irrecoverable error. However, graph is probed for errors in Read loop, meaning - we only check for errors in-between subsequent client requests. Will add it to documentation.

Reply:
Added